object/check.go

```diff
@@ -273,6 +273,10 @@ func CheckPasswordComplexity(user *User, password string) string {
 	return CheckPasswordComplexityByOrg(organization, password)
 }
 
+func CheckLdapUserPassword(user *User, password string, lang string) error {
```

Comment:
you can change `checkLdapUserPassword` to `CheckLdapUserPassword` instead of create a new function

Reply:
Has been adjusted.